server.js

```diff
@@ -73,5 +79,7 @@ oidc.Client.Schema.prototype.invalidate = function invalidate(message, code) {
 
 const app = new Koa();
 app.use(mount(prefix, oidc.app));
-
+if (protocol === 'https') {
+  app.proxy = true;
```

Comment:
we should make this the default, and remove the need to prefix or specify the protocol by default. Still allow to specify the prefix and the protocol to override the x-forwarded-* headers though